README.md

```diff
@@ -43,7 +43,7 @@ To deploy the application:
    application](https://developers.google.com/appengine/docs/python/tools/uploadinganapp) with
 
    ```
-   appcfg.py -A <your-project-id> --oauth2 update .
+   appcfg.py -A <your-project-id> -V v1 update .
```

Comment:
Does this make the next line obsolete? ie if you specify the version, will that necessarily be the default version served by appengine? Or are we assuming that v1 will be the only version?